app/assets/javascripts/modules/locationsearch.js

```diff
@@ -37,6 +37,9 @@ $(document).ready(function() {
       $('.leaflet-container').toggleClass('active');
     });
 
+    // hide zoom control on load
+    $('.leaflet-control-zoom').toggle();
```

Comment:
I'm trying out a minimal look. Not sure how useful those zoom buttons are in this context. Plus there's some weird CSS issues with the buttons after you do a location search that I haven't been able to track down.